src/plugin/analytics/Resources/modules/administration/dashboard/store/reducer.js

```diff
@@ -1,14 +1,16 @@
 
 import {makeInstanceAction} from '#/main/app/store/actions'
 import {makeReducer, combineReducers} from '#/main/app/store/reducer'
+import {makeListReducer} from '#/main/app/content/list/store/reducer'
 import {TOOL_LOAD} from '#/main/core/tool/store/actions'
 
 import {selectors} from '#/plugin/analytics/administration/dashboard/store/selectors'
 
 const reducer = combineReducers({
   count: makeReducer({}, {
     [makeInstanceAction(TOOL_LOAD, selectors.STORE_NAME)]: (state, action) => action.toolData.count
-  })
+  }),
+  list: makeListReducer(selectors.LIST_NAME)
```

Comment:
It should use a more specific name `securityLogs`, because there are lots of lists in this tool.
It is not required for lists to be named `list` in the store.

Reply:
Change done, thx :)